ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/adoptopenjdk_install/tasks/main.yml

```diff
@@ -173,6 +173,36 @@
         path: /tmp/jdk{{ jdk_version }}.tar.gz
         state: absent
 
+# JDK 21 on Linux-riscv64 is a special-case because JDK 21 is the first version that supports
+# RISC-V. There is also no JDK 21 or 20 available on Ubuntu 20.04 that we can use as boot JDK.
+- name: Install JDK {{ jdk_version }} on Linux-riscv64
+  when:
+    - ansible_architecture == "riscv64" and jdk_version == 21
+    - adoptopenjdk_installed.rc != 0
+  tags: adoptopenjdk_install
+    # Api does not return release information for JDK10
+  block:
+    - name: Download jdk{{ jdk_version }} release (Linux-riscv64)
+      get_url:
+        url: https://api.adoptium.net/v3/binary/version/jdk-21.0.1+12.1-ea-beta/linux/riscv64/jdk/hotspot/normal/adoptium
```

Comment:
We should add a checksum into this to verify the download since it's a special case.
For the 21.0.1+12 that means:
```
    checksum: sha256:c9b8b1ca18b13e293688cafbd8990c940ca49104dbeefc242e5c3f8de271abdf
```
and for the 17 download:
```
    checksum: sha256:c9b8b1ca18b13e293688cafbd8990c940ca49104dbeefc242e5c3f8de271abdf
```
These should work in these stanzas in the same as the example at https://github.com/adoptium/infrastructure/blob/7042913997344376ef8a4e6f12a79e6312aa1278/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Ant-Contrib/tasks/main.yml#L34

Reply:
The checksums you've given me here are duplicate. I assume due to a bad c/p. I've added the checksums I get when downloading them locally.